src/grafana/provisioning/dashboards/hipstershop.json

```diff
@@ -811,7 +798,7 @@
       "thresholds": [],
       "timeFrom": null,
       "timeShift": null,
-      "title": "Number of Request by Status",
+      "title": "Number of Response by Status",
```

Comment:
Does y-axis below also need to be updated?

Reply:
yes. good catch.